lib/services/ios-project-service.ts

```diff
@@ -704,7 +704,7 @@ We will now place an empty obsolete compatability white screen LauncScreen.xib f
 		return Promise.resolve();
 	}
 
-	private async mergeInfoPlists(projectData: IProjectData): Promise<void> {
+	private async mergeInfoPlists(release: boolean, projectData: IProjectData): Promise<void> {
```

Comment:
Also I prefer having the projectData as a first argument... but I'm not sure why I prefer it